src/datasets/diabetes.rs

```diff
@@ -7,17 +7,37 @@ use crate::utils::scale;
 /// Ten baseline variables (age, sex, body mass index, average blood pressure, and six blood serum measurements) were obtained for each of n = 442 diabetes patients, as well as the response of interest, a quantitative measure of disease progression one year after baseline.
 #[derive(Clone, Copy, Debug, PartialEq)]
 pub struct Diabetes {
+    /// age in years
     pub age: f32,
+
+    /// sex (two classes, represented as either a 1 or 2)
     pub sex: f32,
```

Comment:
I just noticed that this is an `f32`, and the data themselves are also floating-point values. Should probably be set to an integer, though?